libs/ui/src/seal.tsx

```diff
@@ -31,17 +27,26 @@ const SealContainer = styled.div<{ inverse?: boolean }>`
 `;
 
 export interface SealProps {
+  maxWidth: string;
   seal: string;
   inverse?: boolean;
+  style?: React.CSSProperties;
 }
 
-export function Seal({ seal, inverse }: SealProps): JSX.Element {
+export function Seal({
+  seal,
+  maxWidth,
+  style,
+  inverse,
+}: SealProps): JSX.Element {
   return (
-    <SealContainer
-      inverse={inverse}
+    <SealImage
       aria-hidden
+      src={`data:image/svg+xml;base64,${Buffer.from(seal).toString('base64')}`}
       data-testid="seal"
-      dangerouslySetInnerHTML={{ __html: seal }}
+      alt="Seal"
+      inverse={inverse}
+      style={{ maxWidth, maxHeight: maxWidth, ...(style ?? {}) }}
```

Comment:
Instead of a container with `dangerouslySetInnerHTML`, uses base64 in an `img` component like in VxDesign.